packages/test/test-end-to-end-tests/src/test/serializedContainer.ts

```diff
@@ -0,0 +1,251 @@
+/*!
```

Comment:
i wonder if this would be better checked into https://github.com/microsoft/FluidFrameworkTestData and have the test run as part of the this suite: https://github.com/microsoft/FluidFramework/tree/main/packages/test/snapshots

I'd add a new test in that case. this would also allow us to easily keep multiple copies of the json like the existing snapshot test, so we can ensure we don't break backcompat for old formats

Reply:
Sounds good!  PR into FluidFrameworkTestData here:
https://github.com/microsoft/FluidFrameworkTestData/pull/17